include/micm/solver/state.hpp

```diff
@@ -51,6 +51,12 @@ namespace micm
         const System& system,
         const std::unordered_map<std::string, std::vector<double>>& species_to_concentration);
 
+    /// @brief Set a single species concentration
+    /// @param species the species to set the concentration for
+    /// @param concentration concentration(s) [mol m-3]
+    void SetConcentration(const Species& species, double concentration);
+    void SetConcentration(const Species& species, std::vector<double> concentration);
+
```

Comment:
Do you think we can pass `concentration` as const ref to avoid copy?

Reply:
done!